test/model.index.spec.js

```diff
@@ -172,6 +172,7 @@ describe('Model index function tests', function () {
     });
 
     function checkIndexRes(err, indexRes) {
+      console.dir(err);
```

Comment:
Use inspect if you want to see the whole object. 

I have a handy snippet that turns `inspect<tab>` into

`console.log(require('util').inspect(, { depth: null }));`